static_inventories/gearshift_cluster.yml

```diff
@@ -0,0 +1,77 @@
+---
+all:
+  children:
+    openstack_api:
+      hosts:
+        localhost:
+    jumphost:
+      hosts:
+        airlock:
+          cloud_flavor: 2C-4GB
+    repo:
+      hosts:
+        gs-repo:
+          cloud_flavor: 2C-4GB
+    docs:
+      hosts:
+        docs_on_merlin:
+    sys_admin_interface:
+      hosts:
+        imperator:
+          cloud_flavor: 4C-8GB
+          local_volume_size_extra: 2900
+    deploy_admin_interface:
+      hosts:
+        sugarsnax:
+          cloud_flavor: 4C-8GB
+          local_volume_size_extra: 2900
+    user_interface:
+      hosts:
+        gearshift:
+          cloud_flavor: 24C-240GB
+          slurm_sockets: 2
+          slurm_cores_per_socket: 12
+          slurm_real_memory: 217778
+          slurm_local_disk: 0
+          slurm_features: 'prm03,tmp01'
+          slurm_ethernet_interfaces:
+            - 'vlan983'
+            - 'vlan985.isilon'
+            - 'vlan985.lustre'
+    compute_vm:
+      children:  # Must be item from {{ slurm_partitions }} variable defined in group_vars/{{ stack_name }}/vars.yml
+        regular:
+          hosts:
+            gs-vcompute[01:10]:
+          vars:
+            # flavor_vcompute: htc-node # Quota currently too small for 40 core compute nodes.
```

Comment:
This comment was copy-pasted from a cluster on Merlin cloud where we have this quota issue, but it does not apply to compute nodes on the Shikra cloud where Gearshift is hosted.